modules/mod_ginger_rdf/models/m_dbpedia.erl

```diff
@@ -46,14 +49,66 @@ get_resource(<<"nl.dbpedia.org", _/binary>> = Uri, Context) ->
 get_resource(<<"dbpedia.org", _/binary>> = Uri, Context) ->
     get_resource(Uri, <<>>, Context).
 
+-spec get_resource(Uri::binary(), Language::binary(), z:context()) -> m_rdf:rdf_resource() | undefined.
 get_resource(Uri, Language, Context) ->
-    z_depcache:memo(
-        fun() ->
-            dbpedia:get_resource(<<"http://", Uri/binary>>, Language)
-        end,
-        {Uri, Language},
-        Context
-    ).
+            get_resource_cached(Uri, Language, Context).
+    % z_depcache:memo(
+    %     fun() ->
+    %         get_resource_cached(Uri, Language, Context)
+    %     end,
+    %     {Uri, Language},
+    %     Context
+    % ).
```

Comment:
@mworrell Are you using tkvstore because that is backed by Postgres so will persist between restarts? Or are there other reasons to prefer tkvstore over depcache?

Reply:
Indeed, because of long term caching.

Especially after the months-long problem with nl.dbpedia.org